flame/data.py

```
@@ -112,24 +110,18 @@ def sample(self, indices):
             self.buffer[i] = torch.tensor(self.tokens[start:end], dtype=self.dtype)
         self.token_id = 0
         self.tokens = self.tokens[n_tokens:]
 
-    def randint(
-        self,
-        low: int,
-        high: int,
-        buffer_size: int = 1024,
-        g: torch.Generator = torch.Generator()
-    ) -> Iterable[int]:
+    def randint(self, low: int, high: int, buffer_size: int = 1024, g: torch.Generator = torch.Generator()) -> Iterable[int]:
```

Ruff (0.8.2)

Do not perform function call `torch.Generator` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)